docs/Sample Queries/aws_ec2.md

```diff
@@ -13,20 +13,20 @@ Install the AWS EC2 module
 SELECT * FROM iasql_install('aws_ec2');
 ```
 
-Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `ami-0892d3c7ee96c0bf7` is the AMI ID for Ubuntu in `us-west-2`. Follow this [guide](/ami) to get the latest AMI ID for a given AMI image and AWS region.
+Create two new EC2 instances associated with the `default` security group within a transaction. A instance `name` tag is required. `resolve:ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id` resolves to the AMI ID for Ubuntu in the corresponding AWS region.
 
 ```sql
 BEGIN;
   INSERT INTO instance (ami, instance_type, tags)
-    VALUES ('ami-0892d3c7ee96c0bf7', 't2.micro', '{"name":"i-1"}');
+    VALUES ('resolve:ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id', 't2.micro', '{"name":"i-1"}');
```

Comment:
This is both better and worse. Better in that you don't have to care about what region you're in now, but worse in the "look at the size of that thing" sense.

Reply:
Yep. Unfortunately it increases the length of the query significantly